source/configuration/modules/omfile.rst

```diff
@@ -444,86 +451,57 @@ Caveats/Known Bugs
    atomically closed and in sync. HUPing only after a subset of files
    have been moved results in inconsistencies and will most probably
    render the file set unusable.
+   
+   
 
-Example
--------
+=============  ==============
+sig.provider:  "providername"
+Default:       none
+=============  ==============
 
-The following command writes all syslog messages into a file.
+Selects a signature provider for **log signing**. By selecting a provider, the signature feature is turned on.
 
-::
+Currently, there are two providers available ":doc:`gt <sigprov_gt>`" and ":doc:`ksi <sigprov_ksi>`". 
 
-  action(type="omfile" dirCreateMode="0700" FileCreateMode="0644"
-         File="/var/log/messages")
+**TODO `Sign log messages through signature provider Guardtime <http://www.rsyslog.com/how-to-sign-log-messages-through-signature-provider-guardtime/>`_**
```

Comment:
as noted below, there is only one provider currently implemented, but the infrastructure is designed so that others could be added easily